Source/Utils/WarpXUtil.cpp

```diff
@@ -405,7 +405,7 @@ getWithParser (const amrex::ParmParse& a_pp, char const * const str, float& val)
     // If so, create a parser object and apply it to the value provided by the user.
     std::string str_val;
     Store_parserString(a_pp, str, str_val);
-    val = parseStringtoReal(str_val);
+    val = static_cast<float>(parseStringtoReal(str_val));
```

Comment:
Assuming we update the argument type in the function interface, too:
```suggestion
    val = static_cast<amrex::Real>(parseStringtoReal(str_val));
```